package-res/reportviewer/reportviewer.js

```diff
@@ -1098,6 +1099,15 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
         });
       },
 
+      _hideDialogAndPane: function() {
+        var dlg = registry.byId('feedbackScreen');
+        dlg.hide();
+        if(this._updateReportTimeout >= 0) {
+          this._updateReportTimeout = -1;
```

Comment:
If you remove the handle you got from the "setTimeOut" function call, does that mean you want to cancel that timeout? In that case, wouldn't you want to call "clearTimeOut(..)" as well?